sieve.py

```diff
@@ -572,14 +572,18 @@ def run(
             )
 
             result_map = {
-                "operator": project,
-                "test": test,
-                "test_config": test_config_to_use,
-                "mode": mode,
-                "duration": time.time() - s,
-                "alarm": alarm,
-                "bug_report": bug_report,
-                "test_config_content": open(test_config_to_use).read(),
+                project: {
+                    test: {
+                        test_config:{
+                            mode: {
```

Comment:
I think `mode` should be a layer higher than `test_config`

Reply:
Nice catch. Just fixed